CAP/gap/Derivations.gd

```diff
@@ -41,15 +41,18 @@ DeclareCategory( "IsDerivedMethod", IsAttributeStoringRep );
 #!  purposes.
 #!  The argument <A>target_op</A> is the operation which
 #!  the derived method implements.
-#!  The argument <A>used_ops_with_multiples</A> contains the name of each
-#!  operation used by the derived method, together with a positive
-#!  integer specifying how many times that operation is used and
-#!  optionally a category getter.
+#!  The argument <A>used_ops_with_multiples</A> contains
+#!  
+#!  * the name of each operation used by the derived method,
+#!  * together with a positive integer specifying how many times that operation is used and
+#!  * either a category getter or <C>fail</C>.
+#!  
 #!  This is given as a list of lists, where each sublist has as
 #!  first entry the name of an operation, as second entry an integer and as
-#!  third entry optionally a function. This function should accept the
-#!  category and return a category for which the operation in the first
-#!  entry must be installed for the derivation to be considered valid.
+#!  third entry either a function or <C>fail</C>. This function should accept the
+#!  category for which this derivation will be installed,
+#!  and return a category for which the operation in the first
+#!  entry is installed.
```

Comment:
```suggestion
#!  entry must be installed for the derivation to be considered applicable.
```
It's okay if the operation in the first entry is not installed, then the derivation simply will not be applied.